### ISSUE TITLE
Domain specific layer lists

### ISSUE THREAD

**Issue by janodvarko** [firebug/pixel-perfect#40]
It should be possible to create list of layers for specific domains.

Honza

**Comment by mbuckley** [comment]
@janodvarko Agreed that this would be a nice feature. We could introduce the idea of "collections" to let users keep things organized and give users flexibility around how they create the lists.

A few examples of uses might be:

1) Collections per domain
 -- apple.com
 --firebug.com

2) Pages within a site for a more complicated application
 -- mynewapp / public
 -- mynewapp / dashboard
 -- mynewapp / admin
etc.
